package.json

```
@@ -5,10 +5,11 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
+    "db:build": "npm run build && node dist/db/seed/index.js --resolveJsonModule",
```

No need for `--resolveJsonModule` here I think it won't do anything? It's a TS command
```suggestion
    "db:build": "npm run build && node dist/db/seed/index.js",
```